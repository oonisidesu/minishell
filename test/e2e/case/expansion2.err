AA="out/abc    def"
> ""$AA""
BB="     "
> $BB
CC=""
> $CC
> ""$CC
> ""$CC""
AA="'abc       def'"
$AA
"$AA"
""$AA""
"""$AA"""
F1="   "
$F1
"$F1"
F2=""
$F2
""$F2""
"$F2"
""

