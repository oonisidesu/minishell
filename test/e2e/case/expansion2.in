AA="abc    def"
echo $AA
echo "$AA"
echo ""$AA
AA="'abc    def'"
echo $AA
echo "$AA"
echo ""$AA
AA='"abc    def"'
echo $AA
echo "$AA"
echo ""$AA
BB=$AA
CC="$AA"
DD=""$AA
echo "$BB"
echo "$CC"
echo "$DD"
E1="   -a  "
ls$E1
E2="  ls -a  "
$E2
E3="         "
$E3
