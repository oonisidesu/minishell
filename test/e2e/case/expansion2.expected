minishell: ""$AA"": ambiguous redirect
minishell: $BB: ambiguous redirect
minishell: $CC: ambiguous redirect
minishell: : No such file or directory
minishell: : No such file or directory
minishell: 'abc: command not found
minishell: 'abc       def': command not found
minishell: 'abc: command not found
minishell: 'abc       def': command not found
minishell:    : command not found
minishell: : command not found
minishell: : command not found
minishell: : command not found
